core/express-server/src/ExpressServer.ts

```diff
@@ -14,6 +14,10 @@ import { BentleyCloudRpcConfiguration, RpcConfiguration, WebAppRpcProtocol } fro
  * @public
  */
 export class IModelJsExpressServer {
+  public static defaults = {
```

Comment:
@swbsi could we make this a member of the instance instead of a static on IModelJsExpressServer itself and pass in the default via the constructor?  It'll make the express server easier to test from the app side.

Reply:
Sure, makes sense.